system-test/logging.ts

```diff
@@ -186,6 +189,7 @@ describe('Logging', async () => {
 
     it('should create a sink with a Dataset destination', async () => {
       const sink = logging.sink(generateName());
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
instead of the eslint disable, could have refactored the next line as `const [, apiResponse] = ...` (omit the `_`)

Reply:
thanks for the suggestion! Checking the fix out!